src/commands/delete.ts

```diff
@@ -6,26 +6,27 @@ import listProjects from "../requests/listProjects.js";
 import { getAuthToken } from "../utils/accounts.js";
 import { askQuestion } from "../utils/prompt.js";
 import { GenezioDeleteOptions } from "../models/commandOptions.js";
+import deleteStage from "../requests/deleteStage.js";
 
-export async function deleteCommand(projectId: string, options: GenezioDeleteOptions) {
+export async function deleteCommand(projectId: string | undefined, options: GenezioDeleteOptions) {
     // check if user is logged in
     const authToken = await getAuthToken();
     if (!authToken) {
         throw new UserError(GENEZIO_NOT_AUTH_ERROR_MSG);
     }
 
-    const result = await deleteProjectHandler(projectId, options.force);
-
-    if (result) {
-        log.info("Your project has been deleted");
-    } else {
-        log.info("Your project has not been deleted. Please try again later.");
-    }
+    await deleteProjectHandler(projectId, options.force, options.stage).catch((error) => {
+        log.error(`Your project could not be deleted: ${error.message}`);
```

Comment:
Should we let `genezio.ts` handle the errors instead?

Reply:
Done ✅